superset/utils/core.py

```diff
@@ -946,7 +946,7 @@ def get_since_until(time_range: Optional[str] = None,
 
     """
     separator = ' : '
-    today = parse_human_datetime('today')
+    today = parse_human_datetime('now')
```

Comment:
Tagging @betodealmeida as he wrote most of this method and did the original dynamic time implementation. 

There are clear challenges here. First depending on using a `ts` vs `ds`, a user may want different behaviors, and that's not mentioning the issues around time zones.

Looks like the code here assumes a `ds` and tries to get 7 full days, and @agrawaldevesh , it looks like you are using `ts` and want true relative time.

It seems like Superset needs to know more about the grain of your time column, and maybe needs to offer better/smarter options around relative time filters.

In the meantime, you can use the relative time expressions as shown here: 
<img width="368" alt="screen shot 2019-01-22 at 10 55 57 am" src="https://user-images.githubusercontent.com/487433/51558461-584bb500-1e34-11e9-9c5e-8bc5684d3330.png">

I think 7 days ago may truncate though. Also we've been talking about improving this component so that it shows what the relative expression compute to, right there in that popover.

Reply:
Okay, I think having users remember to use 'now' (instead of default 'today') would be a problem for my deployment of superset. So I will try to make this be configurable instead such that it preserves backward compatibility.

Reply:
I think I agree this should be `now`, and left bound should probably be relative to now as well, @betodealmeida  what do you think?